### PR TITLE
Cm 993 user b learn more climate impact card

### PR DESCRIPTION
**Pull request by johols**
Showing details when clicking "Learn More" by fetching corresponding impact / solution from backend. This PR solves CM-993 and CM-994.

impacts:

image

and solutions:

image

### CODE REVIEW

**Comment by rodriguesk** [comment]
@johols does the impact card overlay have the personal value tags at the bottom? I couldn’t tell from the image you attached. The designs in the story show the personal value tags at the bottom of the overlay for the impact cards.

**Comment by johols** [comment]
> @johols does the impact card overlay have the personal value tags at the bottom? I couldn’t tell from the image you attached. The designs in the story show the personal value tags at the bottom of the overlay for the impact cards. So we want to ensure they are there too. I haven’t checked yet for your code in this PR.

**Comment by johols** [comment]
@rodriguesk Yes, they are present according to the design:
image

**Comment by rodriguesk** [comment]
Awesome! Great work!

File: src/pages/userB/SharedSolutions/SharedSolutions.tsx [hunk 1]
```diff
@@ -43,35 +47,41 @@ import {
...
   const SharedSolutionsOverlay: React.FC<SharedSolutionsOverlayProps> = ({
-    imageUrl,
-    description,
-    sources,
+    solutionIri,
     selectAction,
   }) => {
+
+    const { data, isSuccess } = useQuery(['solutionDetails', solutionIri], () => {
+      if(solutionIri) {
```
Comment (joaorpereira) [inline src/pages/userB/SharedSolutions/SharedSolutions.tsx:60]: !!solutionIri && getSolutionDetails(solutionIri)

File: src/pages/userB/SharedSolutions/SharedSolutions.tsx [hunk 2]
```diff
@@ -81,15 +91,51 @@ import {
...
+      }
+      if(!e.target.checked){  // remove from selected solutions
+        setSolutionIds(solutionIds.filter(item => item.solutionId !== solutionId));
+      }
       // TODO: add select logic
-      console.log('topic selected');
     };
+
+    const isCheckboxDisabled = (currentSolutionId: string) => {
+      if(solutionIds.length < 2 ) {
```
Comment (joaorpereira) [inline src/pages/userB/SharedSolutions/SharedSolutions.tsx:132]:
```
if( solutionIds.length < 2 || solutionIds.find(item => item.solutionId === currentSolutionId)) {
    return false
}
return true
```

File: src/pages/userB/SharedSolutions/SharedSolutions.tsx [hunk 3]
```diff
@@ -81,15 +91,51 @@ import {
...
+
     const handleNextSharing = () => {
-      //TODO: add correct routing
-      push('/path-to-sharing');
+      mutateChooseSharedSolutions.mutate({solutionIds, alignmentScoresId});
     };
   
-    const handleSelectImpact = () => {
+    const handleSelectSolution = (e: React.ChangeEvent<HTMLInputElement>, solutionId: string) => {
+      if(e.target.checked){ // add to selected solutions
```
Comment (joaorpereira) [inline src/pages/userB/SharedSolutions/SharedSolutions.tsx:122]:
```
e.target.checked ? 
setSolutionIds(prevIds => [...prevIds, {solutionId: solutionId}]) : 
setSolutionIds(solutionIds.filter(item => item.solutionId !== solutionId));
```

**Review by joaorpereira — CHANGES_REQUESTED** [review]

**Review by joaorpereira — APPROVED** [review]

**Pull request merged**